libbpf-tools/execsnoop.bpf.c

```diff
@@ -35,22 +36,33 @@ static __always_inline bool valid_uid(uid_t uid) {
 	return uid != INVALID_UID;
 }
 
+#ifdef __TARGET_ARCH_arm64
+SEC("kprobe/do_execveat_common.isra.0")
```

Comment:
This is fragile. Let's handle this difference in userspace.

You can write two BPF programs and enable them when available.

Reply:
You can always use kprobe for aarch64.

Reply:
> Sadly, with kprobe I think we will not be able to get command line arguments.

Why ?

Reply:
I see, it's a struct.